backend/lucene/src/main/java/org/hibernate/search/backend/lucene/scope/LuceneIndexScope.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.lucene.scope;
+
+import java.util.Collections;
+import java.util.Set;
+
+import org.apache.lucene.index.IndexReader;
+
+/**
+ * The {@code LuceneIndexScope} exposes {@link IndexReader}s directly, making it possible to query the Lucene
+ * indexes directly bypassing Hibernate Search.
+ * <p>
+ * The returned IndexReader instances are always read-only and must be closed
```

Comment:
I'm not sure why an index reader would *not* be read-only. That must be referring to some dodgy feature that was removed from Lucene.

```suggestion
 * The returned IndexReader instances must be closed
```